package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "to-do-list",
+  "version": "1.0.0",
+  "description": "This is To do list",
+  "private": true,
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
```

Comment:
- [ ] We need to use at least 2 scripts here, one for development and another one for production. Each one should inform in each mode Webpack should run (development and production).